### ISSUE TITLE
Past events should display timestamps in team calendar view

### ISSUE THREAD

**Issue by lkocman** [openSUSE/calendar-o-o#60]
I think we generally want to show timestamps for past events in team/calendar view, but it should be clear that the event already happened. It shouldn’t rely only on visuals—we need to keep accessibility in mind too.

https://calendar.opensuse.org/teams/product-schedule
